students/14th/eommoonjoo/js/login.js

```diff
@@ -0,0 +1,31 @@
+document.getElementById('logInBtn').onclick = function() {
+    let logInId = document.getElementById('logInId').value;
+    let logInPw = document.getElementById('logInPw').value;
+    if ( logInId === "a" && logInPw === "a") {
+        alert("로그인 성공!");
+    } else {
+        alert("로그인 실패!");
+    }
```

Comment:
```suggestion
  if (logInId === "a" && logInPw === "a") return alert("로그인 성공!");
  alert("로그인 실패!");
```
- 이런식으로 if문에서 미리 return 해주시면 else문을 쓰지 않아도 됩니다.
- if-else분기문을 지양하도록 권장드리고있는데, 우선은 사용하시면서 리뷰 반영을 통해 차차 익혀주시면 좋을 것 같습니다 :)

Reply:
- else를 안쓸 수 있어서 행복해요~~^^